src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -477,6 +477,8 @@ def _revYangConvert(val):
             vValue = list()
             for v in value:
                 vValue.append(_revYangConvert(v))
+        elif leafDict[key]['type']['@name'] == 'boolean':
```

Comment:
We need similar handling while converting from config DB to sonic Yang as well right, in below function:
_findYangTypedValue(self, key, value, leafDict)

Reply:
Hi Praveen,
For _findYangTypedValue(...),
def _findYangTypedValue(self, key, value, leafDict):

#```
        # convert config DB string to yang Type
        def _yangConvert(val):
            # Convert everything to string
            val = str(val)
            # find type of this key from yang leaf
            type = leafDict[key]['type']['@name']

            if 'uint' in type:
                vValue = int(val, 10)
            # TODO: find type of leafref from schema node
            elif 'leafref' in type:
                vValue = val
            #TODO: find type in sonic-head, as of now, all are enumeration
            elif 'head:' in type:
                vValue = val
            else:
                vValue = val
            return vValue
```
```
   when type is 'boolean', val itself is already 'true'. Please see below,
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> PDB set_trace (IO-capturing turned off) >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> 
> /sonic/src/sonic-yang-mgmt/sonic_yang_ext.py(252)_yangConvert()
-> vValue = val
(Pdb) p key
'mgmtVrfEnabled'
(Pdb) p value
*** NameError: name 'value' is not defined
(Pdb) p leafDict
{'mgmtVrfEnabled': OrderedDict([('@name', 'mgmtVrfEnabled'), ('type', OrderedDict([('@name', 'boolean')])), ('default', OrderedDict([('@value', 'false')])), ('__isleafList', False)])}
(Pdb) p val
'true'
(Pdb) 

```

So here the change is not needed.